ui/preference/list_preference.go

```diff
@@ -44,17 +48,23 @@ func NewListPreference(wallet *wallet.Wallet, theme *decredmaterial.Theme, prefe
 		wallet:        wallet,
 		preferenceKey: preferenceKey,
 		defaultValue:  defaultValue,
-		theme:         theme,
+		theme:         l.Theme,
 
 		items:    items,
 		itemKeys: sortedKeys,
 
 		IsShowing: false,
 
-		clickable:         theme.NewClickable(false),
+		clickable:         l.Theme.NewClickable(false),
 		optionsRadioGroup: new(widget.Enum),
+		modal:             l.Theme.Modal(),
 	}
 
+	lp.cancelButton = l.Theme.PlainIconButton(cancelIcon)
```

Comment:
```suggestion
	lp.cancelButton = l.Theme.PlainIconButton(l.Icons.ContentClear)
```
this prevents having to send one icon twice to the same method from the settings page.